### PR TITLE
Update generating_android_project_windows.md

### LINKED ISSUE #1178: [ISSUE][DOCS] Fix style issues | Generating Android Projects on Windows
## Describe the issue

There are a couple of issues in this [page](http://o3de.org/docs/user-guide/platforms/android/generating_android_project_windows/) where the text doesn't adhere to the Style Guide. For example, for #9 and #10 in **Prerequisites**, the paths should be in `code-style` (wrapped with the **`** character). 

## Impacted page(s)

http://o3de.org/docs/user-guide/platforms/android/generating_android_project_windows/

## Expected experience

Review the style guide: https://o3de.org/docs/contributing/to-docs/style-guide/

### PR DESCRIPTION
**Pull request by bhaskar1001101**
## Change summary

Closes #1178
Added backticks to **8.** and **9.** in the **Prerequisites** section on [this page](https://www.o3de.org/docs/user-guide/platforms/android/generating_android_project_windows/)

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [Style Guide](https://o3de.org/docs/contributing/to-docs/style-guide/quick-reference)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Review by willihay — CHANGES_REQUESTED** [review]

**Comment by chanmosq** [comment]
@bhaskar1001101 Can you retarget this PR to development instead of main? You may need to make sure you're branch is up-to-date with main instead of development

**Comment by bhaskar1001101** [comment]
@chanmosq I have retargeted the PR to the development branch. My branch is up-to-date with the main branch.

**Comment by chanmosq** [comment]
@bhaskar1001101 Because the changes are minor, I'll switch it back to `main`, and separately sync it with `development`. This one can be good to merge. Thanks for the fix!

**Review by chanmosq — APPROVED** [review]

**Review by FiniteStateGit — APPROVED** [review]

File: content/docs/user-guide/platforms/android/generating_android_project_windows.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ The following instructions assume that you have:
 5.  CMake {{< versions/cmake >}} or newer is installed and on the system path
 6.  Ninja build system is installed and on the system path
 7.  Git is installed
-8.  [O3DE](https://github.com/o3de/o3de.git) has been cloned locally on the system (to D:\\github\\o3de)
-9.  [Atom Sample Viewer](https://github.com/o3de/o3de-atom-sampleviewer.git) has been cloned locally on the system (to D:\\github\\o3de-atom-sampleviewer)
+8.  [O3DE](https://github.com/o3de/o3de.git) has been cloned locally on the system (to `D:\\github\\o3de`)
+9.  [Atom Sample Viewer](https://github.com/o3de/o3de-atom-sampleviewer.git) has been cloned locally on the system (to `D:\\github\\o3de-atom-sampleviewer`)
```
Comment (willihay) [inline content/docs/user-guide/platforms/android/generating_android_project_windows.md:25]: Whoops, just realized the double backslashes are no longer needed, now that you've got the back quotes in there.
```suggestion
8.  [O3DE](https://github.com/o3de/o3de.git) has been cloned locally on the system (to `D:\github\o3de`)
9.  [Atom Sample Viewer](https://github.com/o3de/o3de-atom-sampleviewer.git) has been cloned locally on the system (to `D:\github\o3de-atom-sampleviewer`)
```

**Review by willihay — APPROVED** [review]

**Pull request merged**